docs/randomblind.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Randomblind issuance
+---
+
+<script type="text/x-mathjax-config">
+  MathJax.Hub.Config({
+    extensions: ["tex2jax.js"],
+    jax: ["input/TeX", "output/HTML-CSS"],
+    tex2jax: {
+      inlineMath: [ ['$','$'], ["\\(","\\)"] ],
+      displayMath: [ ['$$','$$'], ["\\[","\\]"] ],
+      processEscapes: true
+    },
+    "HTML-CSS": { fonts: ["TeX"] }
+  });
+</script>
+<script type="text/javascript" async src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js"></script>
+
+*Randomblind* issuance is a new IRMA feature introduced in
+[irmago v0.6.0](https://github.com/privacybydesign/irmago/releases/tag/v0.6.0)
+that can give the user extra privacy
+guarantees during issuance.  Specifically, if this mode of issuance is enabled
+for an attribute within a credential type, this attribute is guaranteed to 
+  1) be random and 
+  2) remain unknown to the issuer. 
+
+This page explains how to enable this feature and later how this guarantee is
+upheld cryptographically.  The use case that sparked this feature is online
+voting, where voting secrecy plays a vital role. Since it is easier to
+explain by example, we'll use this use case as a practical example.
+
+
+## API
+
+Randomblind issuance is enabled in the scheme by adding the `Randomblind` XML
+tag to an attribute type within the issue specification of a given credential.
```

Comment:
1. This is not an XML tag, but an attribute on a tag;
2. It is spelled without capital `R`.